.coveragerc

```diff
@@ -0,0 +1,4 @@
+[report]
```

Comment:
not sure how to put this in the pyproject.toml

Reply:
Issue resolved here: https://github.com/nedbat/coveragepy/issues/664
Documentation here: https://coverage.readthedocs.io/en/coverage-5.0/config.html